sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.State.cs

```diff
@@ -110,6 +111,9 @@ private void Run()
         {
             try
             {
+                // Prevents the http operations from being instrumented.
```

Comment:
How should I interpret this comment? (it doesn't seem to be related to HTTP at all)

Reply:
rewrote to be more explicit: `// Suppress the outbound Live Metrics service calls from being collected as dependency telemetry.`